index.html

```diff
@@ -67,20 +66,20 @@
         <h3>Algorithms</h3>
         <section id="algorithms-sink-unavailable">
           <h4>Sink no longer available</h4>
-          <p>The audio device associated with <a  data-link-for="HTMLMediaElement"><code>HTMLMediaElement.sinkId</code></a> may become unavailable, for example if it is unplugged.</p>
-          <p>When the audio device identified by the <a  data-link-for="HTMLMediaElement"><code>sinkId</code></a> attribute is no longer available, the user agent must take no action. For example, if the media element's <a href="http://www.w3.org/TR/html5/embedded-content-0.html#dom-media-paused"><code>paused</code></a> attribute is false when the device identified by the sinkId is no longer available, then playback will continue as normal. Of course, audio will not be rendered because the device to which the media element is attached is unavailable.</p>
+          <p>The audio device identified by a media element's <a  data-link-for="HTMLMediaElement"><code>sinkId</code></a> attribute may become unavailable, for example, if it is unplugged.</p>
+          <p>When the audio device identified by the <a  data-link-for="HTMLMediaElement"><code>sinkId</code></a> attribute is no longer available, the user agent must take no action. For example, if the media element's <a href="http://www.w3.org/TR/html5/embedded-content-0.html#dom-media-paused"><code>paused</code></a> attribute is false when the device identified by the sinkId is no longer available, then playback will continue as normal. In this case, audio will not be rendered because the device to which the media element is attached is unavailable.</p>
```

Comment:
In this case,  the comma after "For example" is correct (I think. English is slippery!)